### PR TITLE
Add a promo for Open Beauty Facts

### PR DESCRIPTION
**Pull request by teolemon**
Add a promo for Open Beauty Facts

### CODE REVIEW

File: po/common/common.pot [hunk 1]
```diff
@@ -810,6 +810,16 @@ msgctxt "footer_wiki_link"
 msgid "https://en.wiki.openfoodfacts.org"
 msgstr ""
 
+# Do not translate
```
Comment (stephanegigandet) [inline po/common/common.pot]: Do not translate "Open Beauty Facts" but do translate "Personal care"... I think "Cosmetics" would be safer, personal care can be too easily mistranslated. "Soin personnel" wouldn't mean much in French for instance.

Comment (teolemon) [reply po/common/common.pot]: Ok for cosmetics. Anectodal evidence shows me that cosmetics looses us 50% of the potential userbase.

File: po/common/common.pot [hunk 2]
```diff
@@ -810,6 +810,16 @@ msgctxt "footer_wiki_link"
...
+msgctxt "footer_obf"
+msgid "Open Beauty Facts - Personal care"
```
Comment (teolemon) [inline po/common/common.pot]:
```suggestion
msgid "Open Beauty Facts - Cosmetics"
```

File: po/common/common.pot (hunk 1 above)
Comment (teolemon) [inline po/common/common.pot]:
```suggestion
# Do not translate "Open Beauty Facts" but do translate "Cosmetics"
```

**Pull request merged**